perceval/components/abstract_component.py

```diff
@@ -113,16 +114,17 @@ def get_parameters(self, all_params: bool = False, expressions = False) -> list[
             If False, returns the raw parameters that make up the expressions only. Default `False`.
         """
         param_list = []
-        for param in self._params.values():
+        for param in self._raw_params.values():
```

Comment:
This method had already the expected behaviour for components. 
Instead of changing its behaviour to fix the behaviour of circuit, which requires adding your `_raw_params` member that can quickly lead to inconsistencies, it would be better to redefine this method in ACircuit that would call this method for each subcomponent and merge the results (just beware that all components don't have this method)